visualization/app/codeCharta/ui/codeMap/__snapshots__/codeMap.render.service.spec.ts.snap

```diff
@@ -4,9 +4,10 @@ exports[`codeMapRenderService getSortedNodes should get sorted Nodes as array 1`
 Array [
   Object {
     "attributes": Object {
-      "unary": 1,
+      "mcc": 0,
+      "rloc": 0,
     },
-    "color": "#666666",
+    "color": "#DDDDDD",
```

Comment:
The color changes because of the available metrics on the node. `#666666` is red and `#DDDDDD` is grey.

Before we had an available metric `unary`, that's why the folder had a color. With the current metric it must be flattened, since it has no available metrics to visualize.
The test itself is actually insufficient. In a real case, the testing data would not only be decorated, but also aggregated. The aggregation is not happening here. I will update the test.